roles/postgresql/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+postgresql_image: quay.io/sclorg/postgresql-13-c9s
+postgresql_container_name: postgresql
+postgresql_network: host
+postgresql_restart_policy: always
```

Comment:
These two variables are unused right now. Do you have plans to use them?
(I don't think we *need* that flexibility right now, but it surely can be handy in the future)

Reply:
this was added with the earlier setup, but i guess we can drop it for now.